README.md

```diff
@@ -1,5 +1,9 @@
 # OpenJS Foundation Standards Working Group
 
+The Standards Working Group actively monitors evolving standards to educate projects about developments that might affect them, and helps projects formulate standards strategies and advance them before standards bodies.
```

Comment:
I like the verbiage, and i think it's fine if this ships 'as is'.. Minor thing though: Currently it says educate projects, but I think it would be great if it said something explicitly about how this can be used to help review and shape those proposals?  For example "actively monitors evolving standards to educate projects about developments that might affect them and get their review and input.'?

Reply:
How about

```diff
 ... that might affect them,
+collects projects' needs,
 and helps projects ...
```

That might make it clear that arrows go both directions, and that the WG aggregates inputs from multiple projects, and can help resolve conflicts between projects' needs.